packages/vscode-extension/src/dependency/DependencyManager.ts

```diff
@@ -460,3 +462,18 @@ export function checkMinDependencyVersionInstalled(dependency: string, minVersio
 export async function checkAndroidEmulatorExists() {
   return fs.existsSync(EMULATOR_BINARY);
 }
+
+export function isExpoRouterProject() {
+  try {
+    const appRoot = getAppRootFolder();
+    const packageJson = require(path.join(appRoot, "package.json"));
+    const hasExpoRouterDependency = Object.values<string>(packageJson.dependencies).some(
+      (dependencies: string) => {
+        return dependencies.includes("expo-router");
+      }
+    );
```

Comment:
```suggestion
    const hasExpoRouter = Object.values<string>(packageJson.dependencies).some((dependency) => dependency === "expo-router");
```

In the future, we can think about creating `maybeRequire` that returns undefined instead of throwing.